site/src/editor/mod.rs

```diff
@@ -1012,12 +1012,12 @@ pub fn Editor<'a>(
                     <button
                         class="info-button"
                         data-title="\
-shift Enter   - Run + Format
- ctrl /       - Toggle line comment
-  alt Up/Down - Swap lines
-shift Delete  - Delete lines
- ctrl Z       - Undo
- ctrl Y       - Redo"
+   shift Enter   - Run + Format
```

Comment:
This will have to go on the previous line, since `\` eats all subsequent whitespace.

Reply:
> since \ eats all subsequent whitespace
Does it do that only on the first line? Since other lines already start with whitespace (like "  alt").

Reply:
Yes, only the first

Reply:
Done